usr/src/cmd/ptools/pfiles/pfiles.c

```diff
@@ -417,6 +419,8 @@ show_fileflags(int flags)
 		(void) strcat(str, "|O_LARGEFILE");
 	if (flags & O_XATTR)
 		(void) strcat(str, "|O_XATTR");
+	if (flags & __FLXPATH)
+		(void) strcat(str, "|__FLXPATH");
```

Comment:
Spoke with Andy and I am going to change this to strlcat instead of bumping our buffer value.  As a follow up I will open a bug to change the rest of these to strlcat and open the corresponding CR against -gate.

Reply:
So what happens when you have ALL the flags and strlcat fails?  That seems like you might lose information that way.